core/src/main/java/org/helsinki/paytrail/util/LogUtil.java

```diff
@@ -0,0 +1,37 @@
+package org.helsinki.paytrail.util;
+
+
+import org.slf4j.Logger;
+
+public class LogUtil {
+    public static void filteredLog(Logger log, String text){
+        // filter possible cardtoken out of logs
+        text = filterJsonElementFromString(
+                text,
+                "token",
+                "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx");
+        log.info(text);
+    }
+
+    public static String filterJsonElementFromString(String stringToFilter, String elementName, String valueReplace){
+        String elementNameString = "\"" + elementName + "\"";
```

Comment:
Wrap this to try catch to fallback for defult functionality, just in case because this is very important.